journal/modernized/01-journal.adoc

```diff
@@ -2682,7 +2719,13 @@ It shall go well with them.`"
 
 About this time I was pretty much at home,
 and the enemy and adversary of the growth and prosperity of Truth in these parts,
-stirred up an informer against us, one John David, alias Pugh, a weaver,
+stirred up an informer^
+footnote:[Because of the political insecurity of the times,
+a law had been passed forbidding all religious gatherings except those taking place at the times and places authorized by the state.
+A monetary reward was offered by the government to any who would disclose the existence of an unauthorized religious meeting (called a conventicle).
```

Comment:
This line and the one above -- when you're adding text to the asciidoc like this, aim to keep the lines approximately the same length (or shorter) than the average lines you see that were split by my line splitter process. Keeping them to that length prevents wrapping, which generally makes the diffs easier to see and understand. So, can you quick split these two lines?

And, do the same for the line in the footnote above that includes the phrase "originally plural".